app/Models/Role.php

```diff
@@ -19,4 +30,14 @@ public function scopeWeb(Builder $query): void
     {
         $query->where('guard_name', 'web');
     }
+
+    public function scopeSuperAdmin(Builder $query): void
+    {
+        $query->where('name', 'super_admin');
+    }
+
+    public function scopeAdmin(Builder $query): void
+    {
+        $query->where('name', 'admin');
+    }
```

Comment:
Because roles are mutable, can we rely on these scopes?

I'll say, perhaps we make the Super Admin real immutable and required. But they **_could_** delete the Admin role, right?

Reply:
The plan is to only use these scopes in initial/local setup steps.